index.html

```diff
@@ -640,21 +640,25 @@
 
 ### Opt-in, Consent, Opt-out, Global Controls {#opt-in-out}
 
-Different procedural mechanisms exist to enable [=people=] to control how they interact
+Different mechanisms exist to enable [=people=] to control how they interact
 with systems in the world. Mechanisms that increase the number of [=purposes=] for which
-their [=data=] is being [=processed=] are referred to as [=opt-in=] or
-<dfn data-lt="opt in|opt-in">consent</dfn>; mechanisms that decrease this number of [=purposes=] are known as
+their [=data=] is being [=processed=] or the amount  their [=data=] is [=processed=]
+are referred to as [=opt-in=] or <dfn data-lt="opt in|opt-in">consent</dfn>; mechanisms
+that decrease this number of [=purposes=] or amount of [=processing=] are known as
 <dfn data-lt="opt out">opt-out</dfn>.
 
 When deployed thoughtfully, these mechanisms can enhance [=people=]'s [=autonomy=]. Often,
 however, they are used as a way to avoid putting in the difficult work of deciding which
 types of [=processing=] are [=appropriate=] and which are not, offloading [=privacy labour=]
 to the people using a system.
 
-In specific cases, [=people=] should be able to [=consent=] to more
-sensitive [=purposes=], such as having their [=identity=] recognised across contexts or
-their reading history shared with a company. The burden of proof on ensuring that informed
-[=consent=] has been obtained needs to be very high in this case.
+In specific cases, [=people=] should be able to [=consent=] to data sharing that would otherwise be restricted,
+such as having their [=identity=] or reading history shared across contexts.
+[=Actors=] need to take care that their users are *informed* when granting this [=consent=] and
+*aware* enough about what's going on that they can know to revoke their consent
+when they want to. They should also ensure that they assess the risk of what *could* be done
+with the data they are collecting or sharing, not just what the stated purpose of that collection
+or sharing is.
```

Comment:
We weren't sure that this last sentence fit in the sentence about consent and informing users since we don't think the users can really be adequately informed about the risk of pure data storage. We discussed dropping the sentence and filing an issue to figure out how to put the content back in the right way.

```suggestion
when they want to.
```